characters.py

```diff
@@ -1,154 +1,158 @@
 import random
 from abc import ABC, abstractmethod
 import names
-import math
 
 import modifiers
 
 
 class Character(ABC):
-    def __init__(self, max_hp, max_mana, defence, magic_immunity, mana_regen, hp_regen, spell_mana_cost,
-                 mana_regen_lvl_up, hp_regen_lvl_up, physical_dmg):
+    lvl = 1
+    exp = 0
+    exp_for_lvl = 500
+    exp_gained = 250
```

Comment:
Nie rozumiem co to za zmienne...
Exp_gained to nie to samo co exp?

Reply:
czyli co? po zabiciu przeciwnika wywoluje sie funkcja sprawdzajaca lvl i dodajaca na tej podstawie expa? moze i mialoby to sens, bo mozna wtedy uzaleznic ile tego expa dostanie w zaleznosci od tego, czy zabity przeciwnik byl slabszy, rowny, czy silniejszy  ; o

Reply:
Ja to widzę raczej tak, że expo zależy tylko od przeciwnika, a nie od tego jak dobry jest bohater, który zdobywa expo. Może od levelu, może od maxhp, może też od klasy..? Ale nie powinno mieć znaczenia jak dobry jest bohater zabijający tylko zabijany.

Reply:
zrobiłęm, nie rozumiesz to nie oceniaj!